webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operate/listview/FlowNodeInstanceForListViewEntity.java

```diff
@@ -197,18 +197,14 @@ public int hashCode() {
         activityId,
         activityState,
         activityType,
-        incidentKeys,
         errorMessage,
         incident,
         jobFailedWithRetriesLeft,
         tenantId,
-        pendingIncident,
         position,
         positionIncident,
         positionJob,
-        joinRelation,
-        startTime,
-        endTime);
+        joinRelation);
```

Comment:
❓ Why we deleted these properties?

Reply:
Yes sorry I talked to @sdorokhova about whether this is ok, since these fields are `@JsonIgnore` they do not appear in elasticsearch and therefore they are not relevant to the equals and hashcode, keeping it in breaks my tests as a record generates an entity with this fields but when we retrieve that same entity from ELS it does not have these fields

Reply:
Wondering why we had them 🤔